test/docfx.Tests/BuildCommandTest.cs

```diff
@@ -169,11 +169,11 @@ public void TestGetConfigWithNoInputAndDocfxJsonExists()
                 new string[]
                 {
                     "",
-                    $"<h1 id=\"hello-test1\" sourcefile=\"{_inputFolder}/test1.md\" sourcelinenumber=\"7\">Hello Test1</h1>",
-                    $"<p sourcefile=\"{_inputFolder}/test1.md\" sourcelinenumber=\"8\">Test XRef: <a class=\"xref\" href=\"test2.html#xref2\">Hello World</a>",
+                    $"<h1 id=\"hello-test1\" sourcefile=\"{_inputFolder}/test1.md\" sourcestartlinenumber=\"7\" sourceendlinenumber=\"7\">Hello Test1</h1>",
+                    $"<p sourcefile=\"{_inputFolder}/test1.md\" sourcestartlinenumber=\"8\" sourceendlinenumber=\"10\">Test XRef: <a class=\"xref\" href=\"test2.html#xref2\">Hello World</a>",
```

Comment:
>sourcestartlinenumber [](start = 64, length = 21)

question: why in code it's sourceStartLineNumber while in test case it's all lowercase